lib/gui/pages/main/controllers/image-selection.js

```diff
@@ -65,6 +73,37 @@ module.exports = function(SupportedFormatsModel, SelectionStateModel, AnalyticsS
       return;
     }
 
+    if (SupportedFormatsModel.seemsWindowsImage(image.path)) {
```

Comment:
What about checking the opposite condition, and returning if true, to avoid indenting the whole block below?

```js
if (!SupportedFormatsModel.seemsWindowsImage(image.path)) {
    return this.finalizeImageSelection(image);
}

AnalyticsService.logEvent('Possibly Windows image', image);

...
```